Website/src/Foundation/BranchPresets/Sitecore.Foundation.BranchPresets.csproj

```diff
@@ -0,0 +1,89 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{40613875-C8E6-4784-93F0-298305CE8804}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Sitecore.Foundation.BranchPresets</RootNamespace>
+    <AssemblyName>Sitecore.Foundation.BranchPresets</AssemblyName>
```

Comment:
To follow the website project naming convention, this one should maybe be `Sitecore.Demo.Edge.Foundation.BranchPresets` ?

Reply:
Fixed